zcash_client_backend/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this library adheres to Rust's notion of
 ## [Unreleased]
 
 ### Changed
+- `zcash_client_backend::data_api::AccountBalance`:
+  - Refactored to use `Balance` for transparent funds (issue #1411).
+    `AccountBalance` now has an `unshielded_balance()` that uses `Balance` and replaces the
+    (now deleted) `unshielded` non-negative amount.
```

Comment:
`unshielded` wasn't public as a field (only as a method). So this would be better expressed as "and replaces the tracking of only total unshielded balance.".